appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/reject/reject.controller.js

```diff
@@ -66,7 +68,10 @@ export const postRejectReason = async (request, response) => {
 		return renderSelectReason(request, response);
 	}
 
-	session.rejectIpComment = body;
+	session.rejectIpComment = {
```

Comment:
We have a middleware that does this. It's called addBodyToSession or similar. Have a look in the addComment router